### PR TITLE
Further cherry-picks for the c++20 branch

### PR DESCRIPTION
**Pull request by tkoeppe**
These are changes that either have no effect but decrease the diff between branches, or that are improvements to the overall quality of the final document.

Please discuss.

See https://github.com/cplusplus/draft/pull/4220#issuecomment-697047187 for a list of candidate commits.

### CODE REVIEW

**Comment by tkoeppe** [comment]
Further suggestions:

- "Remove superfluous final \rowsep or \hline in tables."
- "Add missing 'friend' for operator-."

File: source/expressions.tex [hunk 1]
```diff
@@ -2823,8 +2823,8 @@
     typename-specifier \terminal{(} \opt{expression-list} \terminal{)}\br
     simple-type-specifier braced-init-list\br
     typename-specifier braced-init-list\br
-    postfix-expression \opt{\terminal{.} \terminal{template}} id-expression\br
-    postfix-expression \opt{\terminal{->} \terminal{template}} id-expression\br
+    postfix-expression \terminal{.} \opt{\terminal{template}} id-expression\br
+    postfix-expression \terminal{->} \opt{\terminal{template}} id-expression\br
```
Comment (zygoloid) [inline source/expressions.tex:2827]: I omitted this one because it should make no difference to the rendered output.

File: source/lex.tex [hunk 2]
```diff
@@ -1352,8 +1352,8 @@
 alert           &   BEL             &   \tcode{\textbackslash a}                \\
 backslash       &   \textbackslash  &   \tcode{\textbackslash\textbackslash}    \\
 question mark   &   ?               &   \tcode{\textbackslash ?}                \\
-single quote    &   \tcode{'}       &   \tcode{\textbackslash\tcode{'}}         \\
-double quote    &   \tcode{"}       &   \tcode{\textbackslash\tcode{"}}         \\
+single quote    &   \tcode{'}       &   \tcode{\textbackslash '}                \\
+double quote    &   \tcode{"}       &   \tcode{\textbackslash "}                \\
```
Comment (zygoloid) [inline source/lex.tex:1356]: I omitted this one because it should make no difference to the rendered output.

**Review by zygoloid — APPROVED** [review]
I think I'm fine with all of these -- though I don't feel motivated to take some of them, I'm also not opposed. I'm mainly just worried about the sticker shock of people seeing hundreds of changes between DIS and IS, even if many of them are trivial, but we've probably already crossed that bridge.

**Comment by burblebee** [comment]
As for what else from the list that should be added...

These seem like harmless yet nice to have consistency improvements:
```
-[tab:atomic.types.pointer.comp] Fix column captions. (#4137)
-[string.view.template] Wrap synopsis in its namespace (#4038)
-[class.temporary] Omit hyphen from "trivially copyable" (#4002)
-[common.iter.cust] Pluralize subclause heading (#3995)
-[meta.trans.other] "C++ object type" is overly precise (#3969)
-[over.ics.ref] Remove erroneous capitalization (#3972)
```

Several others are changes to the library for which I don't have an
opinion on. Perhaps Casey should review the list?

File: source/classes.tex [hunk 3]
```diff
@@ -728,7 +728,7 @@
 with the same access control\iref{class.access} and
 non-zero size\iref{intro.object}
 are allocated so that later
-members have higher addresses within a class object.
+members have higher addresses within a class object\iref{expr.rel}.
```
[the remark this answers is not in the document]
Comment (tkoeppe) [reply source/classes.tex:731]: Fair enough, I'll just remove the commit!

File: source/expressions.tex (hunk 1 above)
Comment (tkoeppe) [reply source/expressions.tex:2827]: Right, that was precisely a reason in favour for me in order to keep diffs down, but I realize that it makes the Editor's Report more awkward, so I'll remove this and the next commit.

File: source/lex.tex (hunk 2 above)
Comment (tkoeppe) [reply source/lex.tex:1356]: Agreed, see above, removing.

**Comment by tkoeppe** [comment]
I remove the unobservable commits and the one with the questionable xref, and I added the commits for the extraneous table hrules and the missing friends. If this is OK now, I'd like to merge this and then restart a PR with Dawn's suggestions.

**Comment by tkoeppe** [comment]
@CaseyCarter: could you please review the suggestions in https://github.com/cplusplus/draft/pull/4225#issuecomment-697110848?

**Comment by jwakely** [comment]
I assume Dawn meant to get Casey to review the _rest_ of the list that affects the library, not the ones in https://github.com/cplusplus/draft/pull/4225#issuecomment-697110848

Anyway, here's my review of the ones in that comment (green where I feel my opinion counts, i.e. library wording):

-[tab:atomic.types.pointer.comp] Fix column captions. (#4137) :heavy_check_mark: 
-[string.view.template] Wrap synopsis in its namespace (#4038) :heavy_check_mark: 
-[class.temporary] Omit hyphen from "trivially copyable" (#4002) :white_check_mark: 
-[common.iter.cust] Pluralize subclause heading (#3995) :heavy_check_mark: 
-[meta.trans.other] "C++ object type" is overly precise (#3969) :heavy_check_mark: 
-[over.ics.ref] Remove erroneous capitalization (#3972) :white_check_mark:

**Review by jwakely — APPROVED** [review]
None of these seem important to fix, but they are all improvements and so nice to have.

**Comment by tkoeppe** [comment]
You're right, that's probably how it was meant! OK, I'll merge this now and reopen a PR for additional discussions. Thank you!

**Comment by burblebee** [comment]
> I assume Dawn meant to get Casey to review the _rest_ of the list that affects the library, not the ones in [#4225 (comment)](https://github.com/cplusplus/draft/pull/4225#issuecomment-697110848)

Yes, I meant for Casey (or someone from lib) to review [#4220 (comment)](https://github.com/cplusplus/draft/pull/4220#issuecomment-697047187). But given Richard's concerns, I say we stop after this pr.

**Pull request merged**